VulkanHppGenerator.cpp

```diff
@@ -1826,28 +1870,45 @@ void readTypeStruct( tinyxml2::XMLElement * element, VkData & vkData, bool isUni
   it->second.returnedOnly = !!element->Attribute( "returnedonly" );
   it->second.isUnion = isUnion;
 
+  if (element->Attribute("structextends"))
+  {
+    std::string structextends = element->Attribute("structextends");
+    std::vector<std::string> structs;
+    size_t endPos = -1;
+    do
+    {
+      size_t startPos = endPos + 1;
+      endPos = structextends.find(',', startPos);
+      std::string structExtendName = structextends.substr(startPos+2, endPos - startPos - 2);
+      it->second.structExtends.push_back(structExtendName);
+      vkData.extendedStructs.insert(structExtendName);
```

Comment:
This would require a postprocessing step after reading all structs. We do have the link child->parent, but would need the parent->child link since parents are defined after the children.